packages/blade/src/components/Box/styledProps/getStyledProps.ts

```diff
@@ -28,27 +28,27 @@ type StyledPropsInputType = Record<string, any>;
   * ```
   */
 const removeUndefinedStyledProps = (obj: StyledPropsInputType): StyledPropsInputType => {
-  const onlyDefinedStyledProps: StyledProps = {};
+  const onlyDefinedStyledProps: StyledPropsBlade = {};
   for (const k in obj) {
```

Comment:
can we use full names here please 🙈 single chars getting confused with TS things 😭

Reply:
done